src/main/main.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="CheckStyle-IDEA-Module">
+    <option name="configuration">
+      <map />
+    </option>
+  </component>
+</module>
```

Comment:
`.iml` files should not be committed. Also, there should not be an `.iml` file here in the first place, unless you did something you were not supposed to do.